### DISCUSSION TITLE
Pending changes

### DISCUSSION THREAD

**Discussion by Melkjot** [nik-base/auto-tfs#6] [Ideas]
Hi,

I started to use Visual Studio Code for projects development (used Visual Studio) and installed Auto TFS extension to be able to use TFS as SCM. I really like the extension, it provides a lot of commands/features compared to other available TFS extensions on the marketplace.

What I missing are the pending changes and the history.
When I edit/add/file, it would be great to see the changes in Source Control View and to have the possibility to include/exclude files/folders from pending changes.
It would be great also to have the command for check in history of a file/folder/branch. 

Any plans to include mentioned functionalities in next versions?

Thank you!

**Comment by nik-base** [comment]
Hi @Melkjot ,

Pending Changes and include/exclude files and folder features are already implemented. Please make sure you have Auto Sync function ON in extension settings.

There is no command for history currently, but you can use 'Open on server' option, which will open the file/folder in browser in TFS, where you can view history easily.

Also, have a explicit command for History is a good idea, I can consider to implement it in next releases.

Thanks,
Nikhil Patil
